examples/index.html

```diff
@@ -179,9 +179,9 @@ <h4 class="modal-title">Your gif is ready</h4>
               </div>
               <div class="modal-footer">
                 <div>
-                  <div>
+                  <div style="padding: 10px; margin-bottom: 10px;">
                   <p>Framerates</p>
-                    <input type='number' value=1 id="js-download-gif-framerate" />
+                    <input type='number' value=7 id="js-download-gif-framerate" />
```

Comment:
Can you constraint this to be a positive number

Reply:
Done!